liquibase fails: NoClassDefFoundError

## Issue description

Starting liquibase results in:

```
Exception in thread "main" java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
        at liquibase.logging.core.Slf4JLoggerFactory.getLog(Slf4JLoggerFactory.java:9)
        at liquibase.logging.LogService.getLog(LogService.java:39)
        at liquibase.integration.commandline.Main.<clinit>(Main.java:67)
Caused by: java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 3 more
```

### Steps to reproduce

```
nix-shell -p liquibase --run "liquibase --help"
```

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.88, NixOS, 19.03pre162590.562b7a7702c (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(philipp): `""`
 - channels(root): `"nixos-19.03pre162590.562b7a7702c"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

--- a/pkgs/development/tools/database/liquibase/default.nix
+++ b/pkgs/development/tools/database/liquibase/default.nix
@@ -6,7 +6,18 @@ assert mysqlSupport -> mysql_jdbc != null;
 with stdenv.lib;
 let
   extraJars = optional mysqlSupport mysql_jdbc;
-
+  logback-core = fetchurl {
+    url = "http://central.maven.org/maven2/ch/qos/logback/logback-core/1.2.3/logback-core-1.2.3.jar";
+    sha256 = "5946d837fe6f960c02a53eda7a6926ecc3c758bbdd69aa453ee429f858217f22";
+  };
+  logback-classic = fetchurl {
+    url = "http://central.maven.org/maven2/ch/qos/logback/logback-classic/1.2.3/logback-classic-1.2.3.jar";
+    sha256 = "fb53f8539e7fcb8f093a56e138112056ec1dc809ebb020b59d8a36a5ebac37e0";
+  };
+  slf4j = fetchurl {
+    url = "http://central.maven.org/maven2/org/slf4j/slf4j-api/1.7.25/slf4j-api-1.7.25.jar";
+    sha256 = "18c4a0095d5c1da6b817592e767bb23d29dd2f560ad74df75ff3961dbde25b79";
+  };
 in
 
 stdenv.mkDerivation rec {
@@ -34,6 +45,7 @@ stdenv.mkDerivation rec {
     in ''
       mkdir -p $out/{bin,lib,sdk}
       mv ./* $out/
+      cp ${logback-core} ${logback-classic} ${slf4j} $out/lib
 
       # Clean up documentation.
       mkdir -p $out/share/doc/${name}

